src/main/java/org/brapi/test/BrAPITestServer/repository/core/TrialRepository.java

```diff
@@ -21,4 +25,14 @@ Page<TrialEntity> findBySearch(
 			@Param("applyActiveFilter") boolean applyActiveFilter, 
 			@Param("active") boolean active, Pageable pageReq);
 
+	@Modifying
+	@Transactional
+	@Query("UPDATE TrialEntity t SET t.softDeleted = :softDeleted WHERE t.id = :trialId")
+	int updateSoftDeletedStatus(@Param("trialId") String trialId, @Param("softDeleted") boolean softDeleted);
```

Comment:
I'm starting to get the feeling these soft queries could be generified.  Not sure why we need to define them in every entity that needs them.  I suppose maybe it's bc not every entity necessarily has this column, but I think we could employ some kind of repo inheritance for entities that need deletion, this repo could extend from `BrAPIRepository`, and it contains these soft deleted methods generified for any entity that needs them.